jsk_robot_common/jsk_robot_startup/src/quadruped_joystick_teleop.cpp

```diff
@@ -5,6 +5,11 @@
 #include <std_srvs/SetBool.h>
 #include <sound_play/SoundRequest.h>
 
+//#include <algorithm>
+
+template <typename T, size_t N> const T* begin(const T (&a)[N]) { return a; }
```

Comment:
What is the purpose of this function?

Reply:
oh, that is for support multiple buttons to send services, like `button_xxx: [0, 1]`. we do not need anymore, so removed.

Reply:
I see. Thank you.